yb-voyager/cmd/analyzeSchema.go

```diff
@@ -1604,6 +1604,7 @@ func checkPlPgSQLStmtsUsingParser(sqlInfoArr []sqlInfo, fpath string, objType st
 }
 
 func convertIssueInstanceToAnalyzeIssue(issueInstance issue.IssueInstance, fileName string) utils.Issue {
+	summaryMap[issueInstance.ObjectType].invalidCount[issueInstance.ObjectName] = true
```

Comment:
Note for later probably: 
We should have a single function `createAnalyzeSchemaIssue()`  which 
1. creates issue instance
2. adds to summarymap if required.
3. append to schemaAnalysisReport.Issues

`reportCase` today is probably that function, but I see the summaryMap addition being done outside. Do you know if there's any specific reason for doing it that way or just tech debt?

Reply:
Yeah, so we are not doing that in `reportCase` right now because we don't need to mark invalid count for every issue as there can be some ALTER stmts etc.. so we first need a mechanism account these invalid counts as well then probably we can do this in a single place.